microsoft-graph.d.ts

```diff
@@ -36330,7 +36435,7 @@ export namespace SecurityNamespace {
         macAddress?: NullableOption<string>;
         manufacturer?: NullableOption<string>;
         model?: NullableOption<string>;
-        nics?: NullableOption<NicEvidence>;
+        nics?: NullableOption<NicEvidence[]>;
```

Comment:
@andrueastman have the issues addressing this change been completed?

Reply:
@musale This is the one change that is expected. The metadata was previously incorrect as the property should be collection.